model/src/main/kotlin/utils/Extensions.kt

```diff
@@ -208,6 +208,7 @@ fun Identifier.getPurlType() =
         "npm" -> PurlType.NPM
         "nuget" -> PurlType.NUGET
         "pypi" -> PurlType.PYPI
+        "cocoapods" -> PurlType.COCOAPODS
```

Comment:
This needs to map `"pod"` (lowercase name of ORT's internal type) to `PurlType.COCOAPODS`. And please sort this in alphabetically before the `"pypi"` line.

Reply:
I'm sorry, I just understood.